feg/gateway/services/envoy_controller/servicers/envoy_controller.go

```diff
@@ -55,7 +54,7 @@ func (s *envoyControllerService) DeactivateUEHeaderEnrichment(
 		err error
 	)
 	glog.Infof("DeactivateUEHeaderEnrichmentResult received")
```

Comment:
Should we log some identifying information about the request? maybe IP? or subscriber ID?